logicmonitor/provider.go

```diff
@@ -31,10 +31,11 @@ func Provider() *schema.Provider {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     false,
-				Description: "(Experimental) true if going for bulk resource, default is false",
+				Description: "true if going for bulk resource, default is false",
```

Comment:
Are we intentionally removing the `experimental` option?

Reply:
No , added it back.